apps/settings/locales/settings.en-US.properties

```diff
@@ -303,7 +303,7 @@ message-read-reports-details = Request read report for each message sent
 # force the proper directionality of parentheses at the end of the string
 # regardless of the direction of the surrounding layout. Remove them only
 # if the translation puts them between other text.
-mmsTitle.innerHTML= &#x202a;Multimedia Messaging (MMS)&#x202c;
+mmsTitle= &#x202a;Multimedia Messaging (MMS)&#x202c;
```

Comment:
AFAIR, this was an edge case in which the translation of this string was missing in Arabic and on buildtime we would fall back to English.  The parenthesis would take the directionality of the surrounding Arabic text.

Now that I think about this, it almost seems like the best solution would be to have L20n's HTML bindings do something like this:

```javascript
const sheet = doc.head.appendChild(document.createElement('style')).sheet;
sheet.insertRule('[data-l10n-id] { unicode-bidi: isolate; }', 0);
```